src/design-system/InputText.tsx

```diff
@@ -56,7 +73,9 @@ const InputText: React.FC<Props> = (props) => {
           ref={nameInput}
           value={inputValue ?? ""}
           headerStyle={!!props.headerStyle}
-          placeholder={props.valuePlaceholder ?? props.labelPlaceholder}
```

Comment:
What did `valuePlaceholder` refer to previously? Was this just a bug?

Reply:
> What did `valuePlaceholder` refer to previously? Was this just a bug?

It's always "undefined" for both Scenario name and Facility name. I'm going to change my code to include the original as my current code might cause issue for other input fields that are using this component